e2e-tests/e2eTest.spec.ts

```diff
@@ -30,15 +30,21 @@ void describe('e2e-tests', () => {
 	})
 	void it('should return statusCode 409 and cache max-age=60 when the SIM information is not in DB', async () => {
 		const req = await fetchData(getRandomICCID())
+		const text = await req.text()
 		const expectedCacheControl = 'public, max-age=60'
 		assert.equal(req.headers.get('cache-control'), expectedCacheControl)
 		assert.equal(req.status, 409)
+		assert.equal(req.headers.get('content-length'), '0')
+		assert.equal(text, '')
```

Comment:
```suggestion
		const expectedCacheControl = 'public, max-age=60'
		assert.equal(req.headers.get('cache-control'), expectedCacheControl)
		assert.equal(req.status, 409)
		assert.equal(req.headers.get('content-length'), '0')
		assert.equal(await req.text(), '')
```

style: move variable declaration close to where it is used. Since variable is not re-used, this can be done directly.